File: src/c/problem set 1/lesson 4 - cash/cash.c
#include <cs50.h>
#include <stdio.h>

void math(int);

int change;
int total_coins;
int count = 0;

int main(void)
{
    do
    {
        change = get_int("Change Owed in Cents: ");
    }
    while (change < 0);

    math(25);
    math(10);
    math(5);
    math(1);

    printf("Coins Given: %i\n", total_coins);
}

void math(int coin)
{
    if (change == 0)
    {
        return;
    }
    count = change / coin;
    total_coins = total_coins + count;
    change = change - coin * count;
}

File: src/python/problem set 6/lesson 3 - sentimental-cash/cash.py
from cs50 import get_float


while True:
    change = get_float("Change: ") * 100
    if change > 0:
        break

total_coins = 0
coin_value = [25, 10, 5, 1]

for i in range(len(coin_value)):
    n = change // coin_value[i]
    total_coins += n
    change -= coin_value[i] * n

print(int(total_coins))
exit(0)
